src/lib.rs

```diff
@@ -49,7 +50,7 @@ impl HnClient {
 
     /// Create a new `HnClient` instance.
     pub fn init() -> reqwest::Result<Self> {
-        let client = reqwest::Client::builder()
+        let client = reqwest::blocking::Client::builder()
```

Comment:
Since the `Client` is already imported diretly, we don't need the prefix anymore:

```suggestion
        let client = Client::builder()
```